manifests/03-rbac-role-ns-openshift-config.yaml

```diff
@@ -13,3 +13,17 @@ rules:
       - get
       - list
       - watch
+---
+kind: Role
```

Comment:
Why is this Role and the RoleBinding needed ?

Reply:
HelmChartRepository instances could refer certificates stored in a configmap and/or secret in `openshift-config` namespace. See [the api](https://github.com/openshift/api/blob/master/helm/v1beta1/types_helm.go#L67-L76). Therefore, we need to allow service account to read them.